RELEASE.md

```diff
@@ -9,6 +9,8 @@ and newer platforms, we have to do the following:
 
 - [ ] In `.circleci/config.yml`, add new platforms and remove obsolete platforms
 - [ ] Make sure that the tip of the `main` branch passes the CI tests.
+- [ ] Ensure that README.md reflects the current state of the project
+  (description and screenshots)
```

Comment:
We actually cover this in the same file, but in a different section: https://github.com/freedomofpress/dangerzone/blob/49b7736cb49d8b03d44c66a3903ba4b9ba1f9171/RELEASE.md#changelog-version-and-signed-git-tag

So, I suggest we drop this line.

Reply:
Actually, I think it would make more sense to keep it in it's original location. We don't know if during the QA procedure we will find visual issue or just need to do some other change that has visual consequences. Doing it in the end might mitigate that risk.